suite/src/utils/composeDbHelpers.ts

```diff
@@ -37,3 +40,26 @@ export async function waitForDocument(
     await delay(1)
   }
 }
+
+/**
+ * Loads a document from a ceramic node with a timeout.
+ * @param ceramicNode : Ceramic client to load teh document from
+ * @param documentId : ID of the document to load
+ * @param timeoutMs : Timeout in milliseconds
+ * @returns 
+ */
+export async function loadDocumentOrTimeout(ceramicNode: CeramicClient, documentId: StreamID, timeoutMs: number) {
```

Comment:
Totally optional nitpick, but a structure like this might be a bit more readable:

```
export async function loadDocumentOrTimeout(ceramicNode: CeramicClient, documentId: StreamID, timeoutMs: number) {
  let now = Date.now()
  const expirationTime = now + timeoutMs
  while (now < expirationTime) {
    try {
      const doc = await ModelInstanceDocument.load(ceramicNode, documentId);
      return doc; 
    } catch (error) {
      console.log(`Error loading document : ${documentId} retrying`, error)
      await delay(10)
      now = Date.now()
    }
  throw new Error('Timeout waiting for document')
  }
}
```

Reply:
appreciate the nitpicks! this does look more readable!